### ISSUE TITLE
i18n: for 1.26

### ISSUE THREAD

**Issue by jdoleary** [jdoleary/Spellmasons#117]

**Comment by jdoleary** [comment]
i18n: openChat
i18n: Update spell copy for changes in #115

**Comment by jdoleary** [comment]
i18n tasks:
- Accessibility menu
- contaminate changes
- corpse decay
- mana steal
- poison onTurnEnd?
- impendingDoom
- sacrifice
- suffocate / tooltip
- teleport

Other tests:
- test: Does split get removed on death? should it?
- QA Chat
- QA New player syncing logic

**Comment by SoulMuncher** [comment]
(Split can only be removed via purify or changing levels, it is persistent through death. You can also split corpses.)

**Comment by jdoleary** [comment]
Keys:
- Accessibility
- If there are any missing accessibility features...
- Witch
- 2 Turns until suffocation
- Teleport
- spell_teleport
- poison tooltip: "poison" "damage"
